### ISSUE TITLE
Could not resolve host: influxdb

### ISSUE THREAD

**Issue by 2crazyzgb** [sthuber90/docker-speedtest#1]
Hi,
I am trying to run docker-speedtest ni RPi3  and I have problem with accessing influxdb:8086 from speedtest container
here is log from docker

```
speedtest    | Speedtest exited with 0
speedtest     | Download: 49.34
speedtest     | Upload: 51.45
speedtest     | Packet Loss: 0.0
speedtest     | Latency: 8.12
speedtest     | Timestamp: 186806140136
speedtest     | Download (49.34 Mbit/s) send returned with 204
speedtest     | curl: (6) Could not resolve host: influxdb
speedtest     | Upload (51.45 Mbit/s) send returned with 000
speedtest     | curl: (6) Could not resolve host: influxdb
speedtest     | Packet Loss (0.0 %) send returned with 000
speedtest     | Latency (8.12 ms) send returned with 204
speedtest     | curl: (6) Could not resolve host: influxdb
speedtest     | Jitter (16.71 ms) send returned with 000
speedtest     | Sleep 900 before next run. -6491260 s remaining
speedtest     | sleep: invalid number '-6491260'
```

**Comment by sthuber90** [comment]
Hi, can you please provide more details? Did you use the docker-compose file from this repository? Can you please provide an output of `docker ps`?

**Comment by 2crazyzgb** [comment]
HI @sthuber90 . 
yes I used docker-compose file and after docker-compose up -d --build I have problem with speedtest container.
Container speedtest  can not resolve http://influxdb:8086
[log.txt](https://github.com/sthuber90/docker-speedtest/files/6181627/log.txt)

**Comment by sthuber90** [comment]
The linking of the influxdb container to the speedtest container is done through Docker built in functionality. To help you I need to know exactly what you did and I need you terminal output for `docker ps`

**Comment by 2crazyzgb** [comment]
I clone project and run docker build with next commands
```
 git clone https://github.com/sthuber90/docker-speedtest.git
 cd docker-speedtest/
 docker-compose  build --no-cache
```

![Screenshot 2021-03-22 at 15 11 07](https://user-images.githubusercontent.com/11678287/112003063-d5f8d180-8b20-11eb-9ad4-c9a2d6114c18.png)

**Comment by sthuber90** [comment]
Thank you for the details. Have you tried pinging inlfuxdb i.e. `ping influxdb`? If this does not, please also check 

1. `ping google.com`
2. `ping 8.8.8.8`

Let me know the output of the above commands, please.

**Comment by 2crazyzgb** [comment]
```
pi@raspberrypi:~/dockers/docker-speedtest $ docker-compose ps
   Name                 Command                   State                Ports
-------------------------------------------------------------------------------------
chronograf   /entrypoint.sh chronograf        Up               0.0.0.0:8888->8888/tcp
influxdb     /entrypoint.sh influxd           Up               0.0.0.0:8086->8086/tcp
speedtest    /bin/sh -c ./init_test_con ...   Up (unhealthy)
pi@raspberrypi:~/dockers/docker-speedtest $
pi@raspberrypi:~/dockers/docker-speedtest $
pi@raspberrypi:~/dockers/docker-speedtest $ docker-compose exec speedtest sh
/opt/speedtest # ping google.com
ping: bad address 'google.com'
/opt/speedtest # ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8): 56 data bytes
ping: clock_gettime(MONOTONIC) failed
/opt/speedtest # ping influxdb
ping: bad address 'influxdb'
```

**Comment by 2crazyzgb** [comment]
Hi @sthuber90 , 
I founded where is problem.
There is problem with alpine:3.13 (version 3.13) I switched to 3.12 and everything works.

**Comment by sthuber90** [comment]
Okay, glad you found it. Are you referring to the problem around https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.13.0#time64_requirements ? If so, you can still use alpine 3.13 but need to update your libseccomp version.

I'll go ahead and close the issue.

**Comment by jmacenka** [comment]
Hmh just saw this closed issue. Had the same issue and came to the same solution during a late night session yesterday... Why not fix it in the repo so people won't run into the same issue again and again? The fix is those additional 4 letters ":3.12" in your Dockerfile. I can make a fork and pull request for it but seems like overkill.

Will you fixe it?
